rust/sbp/src/messages/observation.rs

```diff
@@ -5953,6 +6023,119 @@ impl WireFormat for PackedObsContent {
     }
 }
 
+/// RAIM exclusion
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PackedObsContentRaimExclusion {
+    /// No exclusion
+    NoExclusion = 0,
+
+    /// Measurement was excluded by SPP RAIM, use with care
+    MeasurementWasExcludedBySppRaimUseWithCare = 1,
```

Comment:
Some of the names are pretty long. We could have some manually curated set of replacements. Or we could add a `name`/`identifier` field to the part of the spec that defines bit field values instead of relying on the description

Reply:
We could also drop the message name prefix to the enum names if we moved each message into its own module

Reply:
We have this problem in the C bitfield support too, I suggested the same thing with having a curated or manually specified name-- the problem with that is needing to go through and create these.  Maybe we can add this in later, and only do it for names that are particularly long and awkward, then fallback back to generation of the name if the "curated" name doesn't exist.